addons/binding/org.openhab.binding.xmltv/src/main/java/org/openhab/binding/xmltv/handler/ChannelHandler.java

```diff
@@ -43,7 +40,6 @@
 import org.openhab.binding.xmltv.internal.jaxb.MediaChannel;
 import org.openhab.binding.xmltv.internal.jaxb.Programme;
 import org.openhab.binding.xmltv.internal.jaxb.Tv;
-import org.slf4j.Logger;
```

Comment:
Why did you remove this? And replaced it with `import java.util.logging.Logger;`.